Wikipedia/Code/SchemeHandler/SchemeHandler+ArticleSectionHandler.swift

```diff
@@ -0,0 +1,73 @@
+import Foundation
+
+protocol ArticleSectionHandlerCacheDelegate: class {
+    func article(for key: String) -> MWKArticle?
+    func cacheSectionData(for article: MWKArticle)
+}
+
+
+extension SchemeHandler {
+    final class ArticleSectionHandler: BaseSubHandler {
+        
+        weak var cacheDelegate: ArticleSectionHandlerCacheDelegate?
+        
+        required init(cacheDelegate: ArticleSectionHandlerCacheDelegate) {
+            self.cacheDelegate = cacheDelegate
+        }
+        
+        override class var basePath: String? {
+            return "articleSectionData"
+        }
+        private var basePath: String? {
+            return ArticleSectionHandler.basePath
+        }
```

Comment:
is the private `basePath` returning the overridden `basePath` needed? It compiles for me when I take it out (the private one)

Reply:
good catch, no it's not. I think at some point I was using it in an instance method but missed cleaning this out.